sdk/dns/Azure.ResourceManager.Dns/src/Customization/Models/DnsNaptrData.Serialization.cs

```diff
@@ -82,7 +82,7 @@ void IJsonModel<DnsNaptrRecordData>.Write(Utf8JsonWriter writer, ModelReaderWrit
             }
             if (Optional.IsCollectionDefined(DnsNaptrRecords))
             {
-                writer.WritePropertyName("NaptrRecords"u8);
+                writer.WritePropertyName("NAPTRRecords"u8);
```

Comment:
Please add test for this change

Reply:
Thanks @ArthurMa1978 Added the tests.
I was not able to add the tests as we needed some server side changes deployed.